doc/conf.py

```diff
@@ -310,4 +314,4 @@ def __init__(self, *args, **kwargs):
 # checks if imports work
 from Orange.data import Table
```

Comment:
Could you comment these three imports out? They were here for testing purposes and I forgot to comment them.